### PR TITLE
feat: allow custom namespace for previews

### PR DESCRIPTION
**Pull request by RedMu**
This change allows you to define an environment variable `PREVIEW_NAMESPACE` which is used in place of the standard jx-org-repo-pr convention.  The inspiration for this was to allow a namespace per build, rather than pr

### CODE REVIEW

**Comment by RedMu** [comment]
From looking at the way in which jx preview destroy works, I think this should be ok.  It should only have any effect if you have defined an environment variable PREVIEW_NAMESPACE

File: .github/workflows/jenkins-x-pr.yaml [hunk 1]
```diff
@@ -14,29 +17,41 @@ jobs:
...
       with:
         context: .
         file: ./Dockerfile
-        push: false
-"on":
-  push:
-    branches-ignore:
+        platforms: linux/amd64,linux/arm64
+        push: true
+        build-args: |
```
Comment (ankitm123) [inline .github/workflows/jenkins-x-pr.yaml:48]: can you set tags to pr-debug? No version is set, and latest should come from release branch

**Comment by Danh4** [comment]
@ankitm123 is there anything outstanding for this?  I've been successfully using an image built with this change for about a month now.

**Comment by RedMu** [comment]
Hi @ankitm123, is anything required for this now?  I've been successfully using it in my cluster for about a month without any ill effect

**Review by tomhobson — APPROVED** [review]

**Pull request merged**